option.go

```diff
@@ -350,6 +368,22 @@ func (o *schemaOption) applyToHandler(config *handlerConfig) {
 	config.Schema = o.Schema
 }
 
+type requestInitializerOption struct {
+	Initializer func(spec Spec, message any) error
+}
+
+func (o *requestInitializerOption) applyToHandler(config *handlerConfig) {
+	config.Initializer = o.Initializer
+}
+
+type responseInitializerOption struct {
+	Initializer func(spec Spec, message any) error
+}
```

Comment:
I see the value of two strongly-typed option functions. But the value of two differen implementation types is much less. I think you could collapse these two into a single `initializerOption` type.

Reply:
Unified.